Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -2026,4 +2026,9 @@ Codeunit 4037 "Helper Functions"
         Currency.Validate("Invoice Rounding Type", Currency."Invoice Rounding Type"::Nearest);
         Currency.Insert(true);
     end;
+
+    procedure StringEqualsCaseInsensitive(Text1: Text; Text2: Text): Boolean
```

Comment:
I would put this function inline or rename it to be more descriptive on why this is needed (e.g. we need case insensitive because it is uppercased in BC, then it would be GPStringEqualBCCode or something like this...

Text1 and Text2 could get a bit better name

Reply:
This is meant to be more generic, something that could be reused when needed. The BC Code uppercase scenario is a common one that this could benefit from in both the migration code and in the unit tests.

The use case here is even if the casing is different for the vendor check name, it should still be considered the same name and shouldn't be set to Name 2.